packages/ui-library/changelog.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## 3.2.0
+
+Enhancements:
+
+* Increases the z-index of the AutoSuggest component. [#20349](https://github.com/Yoast/wordpress-seo/pull/20349)
+* Adds a Container to the Modal. It has a Header, Content and Footer. The Content can overflow. [#20546](https://github.com/Yoast/wordpress-seo/pull/20546)
+* Fixes a bug where the Notifications max width and height would not handle scrolling properly. [#20546](https://github.com/Yoast/wordpress-seo/pull/20546)
```

Comment:
Actually a bugfix, but due to the PR doing many things, marked as enhancement.

Reply:
Thanks! Moved it.